docs/content/Getting-Started/user-quickstart-kind.md

```diff
@@ -0,0 +1,276 @@
+          start="<!--create-ks-edge-cluster1-kind-cluster-start-->"
+          end="<!--create-ks-edge-cluster1-kind-cluster-end-->"
+        %}
+
+        {%
+          include-markdown "../common-subs/create-ks-edge-cluster2-kind-cluster.md"
+          start="<!--create-ks-edge-cluster2-kind-cluster-start-->"
+          end="<!--create-ks-edge-cluster2-kind-cluster-end-->"
+        %}
+
+        {%
+          include-markdown "../common-subs/create-files-and-contexts-for-kind-clusters.md"
+          start="<!--create-files-and-contexts-for-kind-clusters-start-->"
+          end="<!--create-files-and-contexts-for-kind-clusters-end-->"
+        %}
+    === "uh oh, error?"
+        if you apply the ingress and then receive an error while waiting:
+          `error: no matching resources found`
+
+        this might mean that you did not wait long enough before issuing the check command. Simply try the check command again.
+   
+#### 1. Deploy the <span class="Space-Bd-BT">KUBESTELLAR</span> Core component  
+{%
+  include-markdown "../common-subs/deploy-your-kubestellar-core-component-kind.md"
+  start="<!--deploy-your-kubestellar-core-component-kind-start-->"
+  end="<!--deploy-your-kubestellar-core-component-kind-end-->"
+%}
+
+#### 2. Install <span class="Space-Bd-BT">KUBESTELLAR</span>'s user commands and kubectl plugins
+!!! tip ""
+    === "install"
+         {%
+           include-markdown "../common-subs/brew-install.md"
+           start="<!--brew-install-start-->"
+           end="<!--brew-install-end-->"
+         %}
+    === "remove"
+         {%
+           include-markdown "../common-subs/brew-remove.md"
+           start="<!--brew-remove-start-->"
+           end="<!--brew-remove-end-->"
+         %}
+
+#### 3. View your <span class="Space-Bd-BT">KUBESTELLAR</span> Core Space environment
+!!! tip ""
+    === "show all available <span class="Space-Bd-BT">KUBESTELLAR</span> Core Spaces"
+         {%
+           include-markdown "../common-subs/kubestellar-show-available-spaces.md"
+           start="<!--kubestellar-show-available-spaces-start-->"
+           end="<!--kubestellar-show-available-spaces-end-->"
+         %}
+    
+    === "uh oh, error?"
+         Did you received the following error:
+         ```Error: Get "https://some_hostname.some_domain_name:{{config.ks_kind_port_num}}/clusters/root/apis/tenancy.kcp.io/v1alpha1/workspaces": dial tcp: lookup some_hostname.some_domain_name on x.x.x.x: no such host``
+
+         A common error occurs if you set your port number to a pre-occupied port number and/or you set your EXTERNAL_HOSTNAME to something other than "localhost" so that you can reach your <span class="Space-Bd-BT">KUBESTELLAR</span> Core from another host, check the following:
+         
+         Check if the port specified in the **ks-core** Kind cluster configuration and the EXTERNAL_PORT helm value are occupied by another application:
```

Comment:
I would say specifically that it is the _host_ port of the Kind cluster to check, because what is configured is a port mapping --- _two_ ports are mentioned in the configuration.